src/ngraph/runtime/plaidml/plaidml_builder.cpp

```diff
@@ -0,0 +1,850 @@
+/*******************************************************************************
+* Copyright 2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "ngraph/runtime/plaidml/plaidml_builder.hpp"
+
```

Comment:
remove this line. Also `<>` headers go before `""`

Reply:
Done.

(FWIW: This pattern comes from the Google coding standards; the idea is that each header should be the first header included by some .cpp file, to ensure that each header is properly self-contained.)